<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNetworkLoggerPanelBase</class>
 <widget class="QgsPanelWidget" name="QgsNetworkLoggerPanelBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>629</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="mToolbar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="mActionRecord"/>
     <addaction name="mActionClear"/>
     <addaction name="mActionSaveLog"/>
     <addaction name="separator"/>
    </widget>
   </item>
   <item>
    <widget class="QgsFilterLineEdit" name="mFilterLineEdit"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="mActionClear">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionDeleteSelected.svg</normaloff>:/images/themes/default/mActionDeleteSelected.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear Log</string>
   </property>
  </action>
  <action name="mActionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionRecord.svg</normaloff>:/images/themes/default/mActionRecord.svg</iconset>
   </property>
   <property name="text">
    <string>Record Log</string>
   </property>
  </action>
  <action name="mActionShowSuccessful">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Successful Requests</string>
   </property>
  </action>
  <action name="mActionShowTimeouts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Timeouts</string>
   </property>
  </action>
  <action name="mActionSaveLog">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionFileSave.svg</normaloff>:/images/themes/default/mActionFileSave.svg</iconset>
   </property>
   <property name="text">
    <string>Save Logâ€¦</string>
   </property>
  </action>
  <action name="mActionShowCached">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Replies Served from Cache</string>
   </property>
   <property name="toolTip">
    <string>Show replies served directly from local cached data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPanelWidget</class>
   <extends>QWidget</extends>
   <header>qgspanelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
